addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -730,15 +732,17 @@ var SnippetEditor = Widget.extend({
         this.selectorLockWithin = new Set();
         const selectorExcludeAncestor = new Set();
 
-        var $element = this.$target.parent();
-        while ($element.length) {
-            var parentEditor = $element.data('snippet-editor');
-            if (parentEditor) {
-                this._customize$Elements = this._customize$Elements
-                    .concat(parentEditor._customize$Elements);
-                break;
+        if (this.allowParentsEditors) {
+            var $element = this.$target.parent();
+            while ($element.length) {
+                var parentEditor = $element.data('snippet-editor');
+                if (parentEditor) {
+                    this._customize$Elements = this._customize$Elements
+                        .concat(parentEditor._customize$Elements);
+                    break;
+                }
+                $element = $element.parent();
```

Comment:
The right plan would just be to not rely on `data('snippet-editor')` and ask the parents (via a trigger_up). But no need to risk fixing that that way for now: we'll see with the OWL refactoring. Doing it that way is nice for now: just add a comment above the `if`. Something like `// TODO should not rely on .data('snippet-editor') but ask parents`